docker/Dockerfile.ci

```diff
@@ -67,34 +67,55 @@ RUN if [ "${BUILD_SYCL_MODULE}" = "ON" ]; then \
         rm -rf /etc/apt/sources.list.d/oneAPI.list; \
     fi
 
-# Dependencies: basic
+# Dependencies: basic and python-build
+# gcc-11 causes a seg fault in tensormap when built in Release mode. Upgrade to
```

Comment:
Have we found the root cause for that or is it really a bug only due to gcc11?

Reply:
Thanks for catching that. I think it was actually the libunwind issue. I've been working with gcc-11 on jammy (22.04) without issues. Reverted.